config/initializers/regex_patterns_email.rb

```diff
@@ -0,0 +1 @@
+VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
vì sao lại tạo file này mà không đưa tất cả vô file Setting như syntax bên dưới  xong validate sửa lại  là dc 
```
validates :email, presence: true, format: { with: Settings.regex.email }
``` 
 
```suggestion
regex:
  email: !ruby/regexp '/\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i'
```